hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieROTablePathFilter.java

```diff
@@ -211,9 +219,10 @@ public boolean accept(Path path) {
         } catch (TableNotFoundException e) {
           // Non-hoodie path, accept it.
           if (LOG.isDebugEnabled()) {
-            LOG.debug(String.format("(1) Caching non-hoodie path under %s \n", folder.toString()));
+            LOG.debug(String.format("(1) Caching non-hoodie path under %s and %s \n",  folder.toString(), baseDir.toString()));
```

Comment:
`path under %s and %s` might confuse users. Maybe, `path under %s with basePath %s` would be more clear